build_tools/github/check_wheels.py

```diff
@@ -17,18 +17,12 @@
 # 2 windows wheel (win32 + wind_amd64)
 n_wheels = 7 * n_python_versions
 
+# plus three more for arm64 on circleci
```

Comment:
```suggestion
# plus three more for linux/arm64 on circleci
```

as I expect that we will be able to add macos/arm64 on github actions soon (with the help of docker/qemu).